src/web/app/page.component.html

```diff
@@ -42,7 +42,9 @@
     </ul>
   </div>
 </nav>
-<tm-notification-banner *ngIf="notificationTargetUser !== NotificationTargetUser.GENERAL" [url]="getUrl()" [notificationTargetUser]="notificationTargetUser"></tm-notification-banner>
+<tm-notification-banner *ngIf="(notificationTargetUser !== NotificationTargetUser.GENERAL) && (isStudent || isInstructor)"
+                        [url]="getUrl()"
+                        [notificationTargetUser]="notificationTargetUser"></tm-notification-banner>
```

Comment:
Is this check `notificationTargetUser !== NotificationTargetUser.GENERAL` still necessary?

Reply:
Thanks for pointing that out. It has been removed.